### PR TITLE
TOOL-667 Fixed async run issue, reverted secret masking writer usage.

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**

### CODE REVIEW

**Comment by richard-bogdan-bitrise** [comment]
Added test case to check async run. Fixed issue, by reverting secret masker filewriter usage from timeoutcmd. Have no better idea (yet) to fix this issue, merge only if it is more important than the masking feature.

File: bitrise.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ workflows:
         title: Run integration tests
         inputs:
         - content: |-
-            #!/usr/bin/env bash
```
Comment (trapacska) [inline bitrise.yml]: pls put this back

Comment (richard-bogdan-bitrise) [reply bitrise.yml]: Accidentally removed, re-added line.

File: _tests/bitrise.yml [hunk 2]
```diff
@@ -58,4 +58,4 @@ workflows:
         - content: |-
             #!/bin/bash
             echo "FAIL!!"
-            exit 1
+            exit 1
```
Comment (trapacska) [inline _tests/bitrise.yml]: pls put that back

File: _tests/integration/async_step_test.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	"testing"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_AsyncStep(t *testing.T) {
+	configPth := "async_step_test_bitrise.yml"
+
+	for _, aTestWFID := range []string{
```
Comment (trapacska) [inline _tests/integration/async_step_test.go]: please remove the for iteration as there is one case now

**Pull request merged**